libctru/include/3ds/services/fs.h

```diff
@@ -135,6 +135,7 @@ typedef enum
 {
 	ARCHIVE_ACTION_COMMIT_SAVE_DATA = 0, ///< Commits save data changes. No inputs/outputs.
 	ARCHIVE_ACTION_GET_TIMESTAMP    = 1, ///< Retrieves a file's last-modified timestamp. In: "u16*, UTF-16 Path", Out: "u64, Time Stamp".
+	ARCHIVE_ACTION_UNKNOWN			= 0x789D, //< Unknown action; calls FSPXI command 0x56. In: "FS_Path instance", Out: "u32[4], Unknown"
```

Comment:
Replace alignment tabs with spaces (tabs are for indentation, spaces are for alignment)